helmfile.d/snippets/defaults.yaml

```diff
@@ -419,6 +419,7 @@ environments:
             enabled: false
             _rawValues: {}
           istio:
+            defaultRevision: "1-26-0"
```

Comment:
I think that this is internal version thus should not be overwriten by customer values. Therfor, I would recommend it to move it under _derived.istio.defaultRevision

Reply:
As in the current implementation, this needs to be dynamic, but I will think about a way to obtain the information from the cluster, instead of relying on the values.